solutions/1833.cpp

```diff
@@ -0,0 +1,20 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Solution {
+ public:
+  int maxIceCream(vector<int>& costs, int coins) {
+    sort(costs.begin(), costs.end());
+
+    int able_to_buy = 0;
+
+    for (int i = 0; i <= costs.size(); i++) {
```

Comment:
i < costs.size()

Reply:
its working on the test cases the website gives

Reply:
but yeah, np ill make the changes

Reply:
in a general case ,you cant acc vector[vector.size()]